jobs/build.sh

```diff
@@ -0,0 +1,2 @@
+rm skelerest.zip
```

Comment:
should this be:
```suggestion
rm build/skelerest.zip
```
?